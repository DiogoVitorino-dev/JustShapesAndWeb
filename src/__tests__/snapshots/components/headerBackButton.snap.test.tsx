import { render } from "@testing-library/react-native";

import AudioSlider from "@/components/settings/audio/audioSlider";

describe("Header Back Button - Snapshot component test", () => {
  it("should render correctly", () => {
    const tree = render(<AudioSlider />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
