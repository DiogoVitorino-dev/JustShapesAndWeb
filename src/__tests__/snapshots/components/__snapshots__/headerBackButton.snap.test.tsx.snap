// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Back Button - Snapshot component test should render correctly 1`] = `
<View
  style={
    [
      undefined,
      [
        {
          "minWidth": 90,
          "overflow": "hidden",
        },
        undefined,
      ],
    ]
  }
>
  <Image
    animatedStyle={
      {
        "value": {},
      }
    }
    collapsable={false}
    forwardedRef={[Function]}
    source={1}
    style={
      {
        "height": "100%",
        "overflow": "hidden",
        "position": "absolute",
        "width": "100%",
      }
    }
  />
  <View
    animatedStyle={
      {
        "value": {
          "height": "100%",
          "overflow": "hidden",
          "position": "absolute",
          "width": 0,
        },
      }
    }
    collapsable={false}
    style={
      {
        "height": "100%",
        "overflow": "hidden",
        "position": "absolute",
        "width": 0,
      }
    }
  >
    <Image
      animatedStyle={
        {
          "value": {
            "height": "100%",
            "width": 0,
          },
        }
      }
      collapsable={false}
      source={1}
      style={
        {
          "height": "100%",
          "width": 0,
        }
      }
    />
  </View>
  <RNCSlider
    disabled={false}
    inverted={false}
    lowerLimit={-9007199254740991}
    maximumTrackTintColor="transparent"
    maximumValue={100}
    minimumTrackTintColor="transparent"
    minimumValue={0}
    onChange={null}
    onRNCSliderAccessibilityAction={null}
    onRNCSliderSlidingComplete={null}
    onRNCSliderSlidingStart={null}
    onRNCSliderValueChange={null}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    step={0}
    style={
      [
        [
          {
            "height": "100%",
            "width": "100%",
          },
          {
            "zIndex": 10,
          },
        ],
        {
          "height": 40,
        },
      ]
    }
    tapToSeek={true}
    thumbTintColor="transparent"
    upperLimit={9007199254740991}
    value={0}
  />
</View>
`;
